modules/svneverever/__main__.py

```diff
@@ -10,6 +10,7 @@
 import os
 import sys
 import time
+import getpass
```

Comment:
There is no CI to respect `.pre-commit-config.yaml` yet, but this project uses isort and `getpass` would need to go first, according to isort.

If you want to see/use it yourself, please do this on the Git clone:
```shell
virtualenv --python=python3 .py3
source .py3/bin/activate
pip install pre-commit
pre-commit install --install-hooks
pre-commit run --all-files
```
[pre-commit](https://pre-commit.com/) will also check/sanitize future commits for you, then. Please give it a try, pre-commit rocks :smile: